src/App.js

```diff
@@ -9,14 +9,15 @@ function App() {
 
   // Customize the code below
   return (
-    <div className="App">
-      <header className="App-header">
-        <h1 className="App-title">Application title</h1>
+    <div className='App'>
+      <header className='App-header'>
+        <h1 className='App-title'>Social media feed</h1>
```

Comment:
Just a note that you can use the `timelineData.person` in the title.

```suggestion
        <h1 className='App-title'>{timelineData.person}'s Social media feed</h1>
```